tutorials/sns-instances-notification-system/index.mdx

```diff
@@ -0,0 +1,299 @@
+    user_data = {
+      cloud-init = <<-EOF
+      #cloud-config
+      write_files:
+        - content: |
+            #!/bin/bash
+            export TOPIC_ARN="${scaleway_mnq_sns_topic.topic.arn}"
+            export AWS_ACCESS_KEY="${scaleway_mnq_sns_credentials.main.access_key}"
+            export AWS_SECRET_KEY="${scaleway_mnq_sns_credentials.main.secret_key}"
+          owner: root:root
+          path: /etc/profile.d/publisher-server_env.sh
+          permissions: '0755'
+      runcmd:
+        - apt-get update && apt-get install -y docker.io
+        - systemctl start docker
+        - systemctl enable docker
+        - chmod +x /etc/profile.d/publisher-server_env.sh
+        # Using 'source' command doesn't work in cloud-init runcmd because each command runs in a separate shell.
+        # this is why environment variables will be passed directly in the docker run command
+        - docker pull rg.fr-par.scw.cloud/sns-x-instance-tutorial/publisher-server:1.0
+        - |
+          . /etc/profile.d/publisher-server_env.sh
+          docker run -d --restart=always \
+            --name publisher-server \
+            -p 8081:8081 \
+            -e TOPIC_ARN=$TOPIC_ARN \
+            -e AWS_ACCESS_KEY=$AWS_ACCESS_KEY \
+            -e AWS_SECRET_KEY=$AWS_SECRET_KEY \
+            rg.fr-par.scw.cloud/sns-x-instance-tutorial/publisher-server:1.0
+    EOF
+    }
+  }
+  ```
+
+7. Check that your `main.tf` file looks like the fully completed example shown on the [GitHub repository](https://github.com/scaleway/sns-x-instances-tutorial)
+
+
+
+# Run and test your setup
+
+We are now ready to run the command to create all of the infrastructure and resources we have defined in our Terraform files, and carry out the additional steps necessary to test that the publisher is indeed pushing alerts which are being received by the subscriber. The web applications that we defined for each Instance (downloaded from a Docker image) allow us to easily view and check what is going on with each Instance.
+1. Run the command `terraform apply`. You are prompted to enter your public SSH key. Execute in your terminal `cat ~/.ssh/`, select with `<tab>` the `*.pub`, paste the result on your local machine to easily access this key, and enter `yes`
+2. From the [Instances section](https://console.scaleway.com/instance/servers) of the Scaleway console, access and copy the public IP address of the Instance we created via Terraform (identifiable from its name: `subscriber_server`).
+3. In a browser, go to home page of your subscriber Instance by inserting the IP address copied in the previous step into the following address: ```http://<your_subscriber_ip_address>:8081```.
+4. Still in the browser, click `Confirm subscription` to confirm tue subscription to the SNS topic we created. If you have an error because the URL hasn't been received, you can reload the page. It should take less than 10 seconds to appear.
```

Comment:
```suggestion
4. Still in the browser, click `Confirm subscription` to confirm the subscription to the SNS topic we created. If you have an error because the URL hasn't been received, you can reload the page. It should take less than 10 seconds to appear.
```